Sources/IO/Misc/PDBReader/index.js

```diff
@@ -0,0 +1,257 @@
+import * as macro from '../../../macro';
```

Comment:
don't forget to register your new class into the index.js of the module directory.